govdelivery/xml_response_parsers.py

```diff
@@ -14,36 +15,45 @@ def topic_xml_as_dict(unparsed_xml, only_listed=False):
     return code_map
 
 
+def listed_topic_xml_as_dict(unparsed_xml):
+    return topic_xml_as_dict(unparsed_xml, only_listed=True)
+
+
 def subscriber_topics_as_list(unparsed_xml):
     root = ET.fromstring(unparsed_xml)
     codes = []
+
     for topic in root.findall('topic'):
         codes.append(topic.find('to-param').text)
 
     return codes
 
 
-def visible_category_xml_as_dict(unparsed_xml):
-    return category_xml_as_dict(unparsed_xml, only_listed=True)
-
+# This parser is currently unused.
+# TODO: Write corresponding get_subscriber_answers client method.
 def subscriber_responses_as_list_of_dicts(unparsed_xml):
     root = ET.fromstring(unparsed_xml)
     responses = []
+
     for response_tag in root.findall('response'):
         answer_id_tag = response_tag.find('answer-id')
 
         question_id = response_tag.find('question-id').text
-        
+
         answer_id = answer_id_tag.text
-        answer_id_is_null = ('nil' in answer_id_tag.attrib and answer_id_tag.attrib['nil'] == 'true')
-        
+        answer_id_is_null = (
```

Comment:
Could this be simplified to `answer_id_tag.attrib.get('nil') == 'true'`?

Reply:
Sure thing. Done in 08023e7.